wagtail/users/utils.py

```diff
@@ -25,26 +27,59 @@ def user_can_delete_user(current_user, user_to_delete):
     return True
 
 
-def get_gravatar_url(email, size=50):
-    default = "mp"
-    size = (
-        int(size) * 2
-    )  # requested at retina size by default and scaled down at point of use with css
+def get_gravatar_url(email, size=50, default_params={"d": "mp"}):
+    """
+    See https://gravatar.com/site/implement/images/ for Gravatar image options
+    Example usage:
+    ```python
+    # Basic usage
+    gravatar_url = get_gravatar_url('user@example.com')
+
+    # Customize size and default image
+    gravatar_url = get_gravatar_url(
+        'user@example.com',
+        size=100,
+        default_params={'d': 'robohash', 'f': 'y'}
+    )
+    Note: If any parameter in `default_params` also exists in the provider URL,
+    It will be overridden by the provider URL's query parameter.
+    ```
```

Comment:
move the note to outside the code example.


````suggestion
    ```

    Note: If any parameter in `default_params` also exists in the provider URL,
    It will be overridden by the provider URL's query parameter.
````

Also Markdown code examples will not work here, they need to be RST.

See a good example here https://github.com/wagtail/wagtail/blob/94d3557cea99a5674397ad7db29529de9dd131c5/wagtail/models/__init__.py#L873-L898